oem/zt/zt.go

```diff
@@ -0,0 +1,96 @@
+package zt
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stmcginnis/gofish/common"
+	"github.com/stmcginnis/gofish/redfish"
+	"io"
+	"net/http"
+)
+
+var (
+	submitTestEventTarget = "/redfish/v1/EventService/Actions/EventService.SendTestEvent"
+)
+
+type (
+	subscriptionPayloadZT struct {
+		Destination string                           `json:"Destination"`
+		HTTPHeaders map[string]string                `json:"HttpHeaders,omitempty"`
+		Protocol    redfish.EventDestinationProtocol `json:"Protocol,omitempty"`
+		Context     string                           `json:"Context,omitempty"`
+	}
+
+	// subscribeResponseType zt uses a unique subscription response.
+	subscribeResponseType struct {
+		OdataContext        string `json:"@odata.context"`
```

Comment:
To keep this consistent with the rest of the library, can you make these "OData" instead of "Odata"?

https://github.com/stmcginnis/gofish/blob/main/common/types.go#L37

Reply:
done